packages/e2e-tests/config/setup-performance-test.js

```diff
@@ -32,13 +34,18 @@ beforeAll( async () => {
 
 	await trashAllPosts();
 	await trashAllPosts( 'wp_block' );
-	await setupBrowser();
+	await clearLocalStorage();
+	await setupPage();
 	await activatePlugin( 'gutenberg-test-plugin-disables-the-css-animations' );
-	await page.emulateMediaFeatures( [
-		{ name: 'prefers-reduced-motion', value: 'reduce' },
-	] );
 } );
 
 afterEach( async () => {
-	await setupBrowser();
+	// Clear localStorage between tests so that the next test starts clean.
+	await clearLocalStorage();
+	// Close the previous page entirely and create a new page, so that the next test
+	// isn't affected by page unload work.
+	await page.close();
```

Comment:
would it be wise to clear local storage after closing the page? I know in some code I've worked with we have periodic flushes to localStorage which I wonder if, on occasion, could write after we clear but before we close the page.

Reply:
I'm not sure if that's possible 🤔 `clearLocalStorage` is a utility that simply sends `window.localStorage.clear()` to the page, so you need to be on a page for this to actually work. We'd need to navigate to a known localStorage-flush-free page in the same origin before running the command.

Reply:
aha. now you're bringing back bad memories, also I think I confused this for the async API with `IndexedDB`